pkg/compliance/clustercompliancereport_test.go

```diff
@@ -35,7 +35,8 @@ func loadResource(filePath string, resource interface{}) error {
 
 var _ = ginkgo.Describe("cluster compliance report", func() {
 	config := etc.Config{
```

Comment:
I suggest that we don't require operator-specific types (especially whole config) in the compliance package.